### PR TITLE
Refactor test script

### PR DESCRIPTION
**Pull request by rokshana-b**
Changed the test script to make code more reusable

### CODE REVIEW

File: test/acceptance/test/smoke_tests.go [hunk 1]
```diff
@@ -26,10 +26,16 @@ func FileExists(name string) bool {
...
+var _ = Describe("WEGO Acceptance Tests", func() {
+
+	var session *gexec.Session
+	var err error
 
 	BeforeEach(func() {
 
+		Context("Given I have a wego binary installed on my local machine", func() {
+			Expect(FileExists(WEGO_BIN_PATH)).To(BeTrue())
+		})
```
Comment (iahmad9) [inline test/acceptance/test/smoke_tests.go:38]: I think Context container is not recommended to be used in BeforeEach or It blocks, instead it is preferred to keep them outside. Any particular reason for using it in this way?

File: test/acceptance/test/smoke_tests.go [hunk 2]
```diff
@@ -26,10 +26,16 @@ func FileExists(name string) bool {
 	return true
 }
 
-var _ = Describe("WEGO Acceptance Tests ", func() {
+var _ = Describe("WEGO Acceptance Tests", func() {
+
+	var session *gexec.Session
+	var err error
```
Comment (iahmad9) [inline test/acceptance/test/smoke_tests.go:32]: Good one 👍

File: test/acceptance/test/smoke_tests.go (hunk 1 above)
Comment (rokshana-b) [reply test/acceptance/test/smoke_tests.go:38]: It was the only way I could make the script run... I wanted to run the 'Given' condition before each test case but the hierarchy of 'Describe', 'Context', 'It', 'By' was giving me problems

Comment (rokshana-b) [reply test/acceptance/test/smoke_tests.go:38]: I've changed `Context` to `By` — seems to work just fine.

**Review by iahmad9 — APPROVED** [review]

File: test/acceptance/test/smoke_tests.go [hunk 3]
```diff
@@ -104,5 +90,43 @@ var _ = Describe("WEGO Acceptance Tests ", func() {
...
+			Expect(err).ShouldNot(HaveOccurred())
+		})
+
+		VerifyUsageText()
+
+	})
+
+	It("Verify that wego flux can print out version information", func() {
+
+		By("When I run 'wego flux -- -version", func() {
```
Comment (iahmad9) [inline test/acceptance/test/smoke_tests.go:122]: would be good if this is changed to `wego flux -- -v` as that's what the test case is testing.

**Pull request merged**